generators/rails-generator/README.md

```diff
@@ -0,0 +1,84 @@
+# rails-generator
+
+This is a generating script for rails gems.
+
+Dynamic method creation, dynamic inheritance, and method delegation in rails are complex.
+
+We aim to automate as much as possible the parts that can be automated, and only write the parts where we want to specify the type.
+
+## Development
+
+### Install
+
+```
+$ bundle install
+$ bundle exec rbs collection install
+```
+
+### Add RBS definition
+
+Update `/known_sig` files.
+
+The directories are separated by gem and by version, and the generator will automatically select the necessary files.
+
+Note that if multiple version directories exist, only one will be selected.
+
+### Add version
+
+Update `Rakefile` and `/gemfiles`.
+
+### Add gem
+
+Update `Rakefile` and `/tasks`.
+
+## Rake tasks
+
+The default task will regenerate all code.
+
+```
+$ bundle exec rake
+```
+
+All subtasks are separated by version.
+
+```
+$ bundle exec rake 6.0:all
+$ bundle exec rake 6.1:generate
+```
+
+Please see `$ bundle exec rake -T` also.
+
+### Generate task
+
+```
+$ bundle exec rake 7.0:generate
+```
+
+Generate RBS file to `out` dir.
+
+Rails gems has dependencies, so it loads all library code and outputs an RBS file.
+
+### Export task
+
+```
+$ bundle exec rake 7.0:active_support:export
+```
+
+From the `out` directory, select the necessary files for each gem and copy them to `export`.
+Generate a test script at the same time so that the entire directory can be replaced.
```

Comment:
Why the `export` directory is necessary? I think it's enough that this task copies the files to `gems/GEM_NAME/` directly and runs the test in the directory.

Reply:
Removed `export` and `validate`.
Also, changed the install task to copy directly to `/gems`.
Also, added a test task that executes `_scripts/test`.